docs/content/index.html

```diff
@@ -15,6 +15,44 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width">
+	
+	<!-- AGLS Metadata -->
+	<link rel="schema.DCTERMS" href="http://purl.org/dc/terms/" />
+	<link rel="schema.AGLSTERMS" href="http://www.agls.gov.au/agls/terms/" />
+	<meta name="DCTERMS.title" content="Content page template - Australian Government Design System" />
+	<meta ... scheme="AGLSTERMS.AglsAgent" content="corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=+61 427 136 791" />
+	<meta name="DCTERMS.publisher" scheme="AGLSTERMS.AglsAgent" content="corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=+61 427 136 791" />
+	<meta name="DCTERMS.description" content="Content pages display information clearly" />
+`	<meta name="DCTERMS.modified" scheme="DCTERMS.ISO8601" content="2010-07-29T12:05:44+10:00" />`
+	<meta name="DCTERMS.subject" content="design system, templates" />	
+	<meta name="DCTERMS.language" scheme="DCTERMS.RFC4646" content="en-AU" />
+	<meta name="DCTERMS.type" content="Content page template" />
+	<meta name="DCTERMS.coverage" content="Australia" />
+	<meta name="DCTERMS.spatial" content="Australia" />
+	<meta name="AGLSTERMS.documentType" content="text" />
+	<meta name="msapplication-TileColor" content="#ffffff" />
+	<meta name="theme-color" content="#ffffff" />
+	<meta name="robots" content="index, follow" />
+	<meta name="author" content="Digital Transformation Agency" />
+	<meta name="description" content="Content pages display information clearly" />
+	
+	<!-- TWITTER CARD -->
+	<meta name="twitter:card" content="summary_large_image" />
+	<meta name="twitter:title" content="Australian Government Design System" />
+	<meta name="twitter:description" content="Content pages display information clearly" /> 
+	<meta name="twitter:site" content="@DTA" />	
+	<meta name="twitter:image" content="../assets/favicons/designsystem.jpg" />
```

Comment:
Bit of a tough one - OG and Twitter image metadata expect content values to be full URLs, complete with protocols. Relative paths aren't officially supported.

https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image